src/App.js

```diff
@@ -4,16 +4,45 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "bootstrap/dist/js/bootstrap.bundle.min";
 import PackageCard from './components/PackageCard';
 import SignInUpPage from './components/SignInUpPage';
-import {Routes, Route} from 'react-router-dom';
+import AdminPage from './components/AdminPage';
+import {Routes, Route, Navigate} from 'react-router-dom';
+import ToursAdmin from './components/ToursAdmin';
+import TravelBuddiesAdmin from './components/TravelBuddiesAdmin';
+import BookingsAdmin from './components/BookingsAdmin';
+import AddTour from './components/AddTour';
+import ApprovedTravelBuddies from './components/ApprovedTravelBuddies';
+import PendingTravelBuddies from './components/PendingTravelBuddies';
+
+
+
 
 
 function App() {
   return (
     <div className="App">
       <Header />
-      <PackageCard />
+      {/* <PackageCard /> */}
       <Routes>
+        <Route path='/' element={<PackageCard
```

Comment:
Here as well

Reply:
I commit the App.js again, is that what you wanted fixed? I am a little confused on what you are asking

Reply:
I don't know why this part is shown as a modification even though this code seems to be in the main branch already. I just wanted to make sure your PR only includes the modifications you made